.github/CODEOWNERS

```diff
@@ -16,10 +16,15 @@
 /modules/core/ @colin-axner @fedekunze @AdityaSripal 
 /proto/ibc/core/ @colin-axner @fedekunze @AdityaSripal 
 
+## CODEOWNERS for core/02-client
+
+/modules/core/02-client @seantking @damiannolan
+/proto/ibc/core/client @seantking @damiannolan
```

Comment:
Does `/modules/core` take a precedence over this or vice versa? 
i.e. Should these lines include the existing codeowners?

Reply:
I was reading the [code owners docs in GitHub](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners) and in theory the code owners in the higher level directory should have also still code ownership on the subdirectories... I can just add the full list of code owners here just in case, though.

Reply:
Looks like this is not true. When I look at the 02-client directory, only @seantking and @damiannolan are code owners. You will need to add all code owners (later declarations take precedence over default owners)